spec/requests/profile/posts/new_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
```

Comment:
There is no API exposure in the gateway so this test should not be a request test.